src/services/urlUtils.ts

```diff
@@ -68,22 +68,18 @@ namespace UrlUtils {
       refinements: urlRefinements = [],
       sort: urlSort = Selectors.sort(store),
     } = state;
-    let request: Partial<Request> = {};
-    const pageSize = Adapters.Request.clampPageSize(page, urlPageSize);
-    const skip = Adapters.Request.extractSkip(page, pageSize);
-    const refinements = <any>urlRefinements.map((refinement) =>
+    let request: Partial<Request> = { collection, query };
```

Comment:
`request` does not get reassigned, so this can be `const`.

Reply:
Confirmed, will do.